prod-sample/sample-modules/frank-lloyd-root/0.0.3/src/pwa.js

```diff
@@ -14,7 +14,46 @@
  * permissions and limitations under the License.
  */
 
+export const defaultSplashScreen = 'http://localhost:3001/static/modules/frank-lloyd-root/0.0.3/assets/pwa-icon-512px.png';
```

Comment:
this might cause some issues when running the remote integration tests

Reply:
Good call, I'll remove them in favor of config/state driven urls.